scripts/write_runtime_constants.py

```diff
@@ -0,0 +1,29 @@
+import json
+from pathlib import Path
+from typing import Union
+
+import tomli
+from starkware.cairo.lang.version import __version__ as cairo_v
+
+from protostar.utils.protostar_directory import RuntimeConstant, ProtostarDirectory
+
+script_path = Path(__file__)
+project_root = Path(script_path).parent.parent
+
+raw_pyproject = (project_root / "pyproject.toml").read_text("utf-8")
+pyproject = tomli.loads(raw_pyproject)
+protostar_v = pyproject["tool"]["poetry"]["version"]
+
+RUNTIME_CONSTANTS: dict[RuntimeConstant, Union[str, int]] = {
```

Comment:
Please define a type for `dict[RuntimeConstant, Union[str, int]]` in the `protostar.utils.protostar_directory`.

Currently, writer accepts `int`s, but the reader cast everything to `str`.

Reply:
done, hardcoded it to str for now